ui/imports/shared/controls/AddressInput.qml

```diff
@@ -21,6 +21,7 @@ Item {
     property bool isResolvedAddress: false
     property int parentWidth
     property bool addContactEnabled: true
+    property alias wrongInputValidationError: contactFieldAndList.wrongInputValidationError
```

Comment:
I might be missing something here, but it feels unnecessary that we're introducing this property in the first place. Isn't this exactly what `validationError` is for?

Reply:
Hey @PascalPrecht so there is a reason why I cant touch the validationError because it changes as per the function. I feel these components need to be relooked at also by using gStatusQ components etc. They are not written super generically as of now and using them elsewhere is causing issues. for now, as this fix is needed for the release this is the solution I found without impacting other places where this component is used. 

I know it isnt the best solution, but for the my speicifc use case, it is needed :(